installer-downloader/Cargo.toml

```
@@ -0,0 +1,61 @@
+[package]
+name = "installer-downloader"
+description = "A secure minimal web installer for the Mullvad app"
+authors.workspace = true
+repository.workspace = true
+license.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+
+[lints]
+workspace = true
+
+[build-dependencies]
+anyhow = "1.0"
+winres = "0.1"
+windows-sys = { version = "0.52.0", features = ["Win32_System", "Win32_System_LibraryLoader", "Win32_System_SystemServices"] }
+
+[dependencies]
+anyhow = "1.0"
+tokio = { version = "1", features = ["full"] }
+async-trait = "0.1"
+
+mullvad-update = { path = "../mullvad-update" }
+
+[target.'cfg(target_os = "windows")'.dependencies]
+windows-sys = { version = "0.52.0", features = ["Win32_System", "Win32_System_LibraryLoader", "Win32_System_SystemServices"] }
+native-windows-gui = { version = "1.0.12", features = ["image-decoder"] }
```

Missing `Win32_Graphics` feature
```suggestion
windows-sys = { version = "0.52.0", features = ["Win32_System", "Win32_System_LibraryLoader", "Win32_System_SystemServices", "Win32_Graphics"] }
```

---

Work for me now
<!-- Reviewable comment -OKRunH489ka5cgA85j_:b8rf29b -->
<!-- Sent from Reviewable.io -->